src/datasources/staking-api/kiln-api.service.ts

```diff
@@ -200,6 +218,14 @@ export class KilnApi implements IStakingApi {
     }
   }
 
+  async clearStakes(safeAddress: `0x${string}`): Promise<void> {
```

Comment:
What do you think about `clearStakesCache`.
nit: It would be nice to have a JSdoc comment here as well.

Reply:
Changed in a2ddb8d0a00494f553f85b88871705788d468006/1a78503e8d98bc3f704be5838685dfbbd1b369ad. If we think it best, we should also update other cache-clearing methods as we don't have `-Cache` elsewhere.

It shouldn't be done here to keep the diff minimal, but I'd subsequently like to move the comments to the class interface as well.